HNC: Setup dev env | Cert Manager release 0.8.1 yaml file uses apps/v1beta1 Deployment

By following the guides to setup development environment. Running make `kind-reset` to stop any existing KIND cluster and setup a new one, including the cert manager required to run the webhooks fails using kind image `kindest/node:v1.16.3`.

* Command to run:

```bash
make kind-reset
```

* Error:

```bash
unable to recognize "https://github.com/jetstack/cert-manager/releases/download/v0.8.1/cert-manager.yaml": no matches for kind "Deployment" in version "apps/v1beta1"
unable to recognize "https://github.com/jetstack/cert-manager/releases/download/v0.8.1/cert-manager.yaml": no matches for kind "Deployment" in version "apps/v1beta1"
unable to recognize "https://github.com/jetstack/cert-manager/releases/download/v0.8.1/cert-manager.yaml": no matches for kind "Deployment" in version "apps/v1beta1"
make: *** [deploy-prereq] Error 1
```

--- a/incubator/hnc/Makefile
+++ b/incubator/hnc/Makefile
@@ -139,7 +139,7 @@ deploy-prereq:
 	@kubectl cluster-info
 	-kubectl create namespace cert-manager
 	kubectl label namespace cert-manager certmanager.k8s.io/disable-validation=true --overwrite
-	kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.8.1/cert-manager.yaml
+	kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml
 
 # Push the docker image
 docker-push: docker-build

--- a/incubator/hnc/config/certmanager/certificate.yaml
+++ b/incubator/hnc/config/certmanager/certificate.yaml
@@ -1,6 +1,6 @@
 # The following manifests contain a self-signed issuer CR and a certificate CR.
 # More document can be found at https://docs.cert-manager.io
-apiVersion: certmanager.k8s.io/v1alpha1
+apiVersion: cert-manager.io/v1alpha2
 kind: Issuer
 metadata:
   name: selfsigned-issuer
@@ -8,7 +8,7 @@ metadata:
 spec:
   selfSigned: {}
 ---
-apiVersion: certmanager.k8s.io/v1alpha1
+apiVersion: cert-manager.io/v1alpha2
 kind: Certificate
 metadata:
   name: serving-cert  # this name should match the one appeared in kustomizeconfig.yaml

--- a/incubator/hnc/config/default/kustomization.yaml
+++ b/incubator/hnc/config/default/kustomization.yaml
@@ -40,16 +40,16 @@ vars:
 - name: CERTIFICATE_NAMESPACE # namespace of the certificate CR
   objref:
     kind: Certificate
-    group: certmanager.k8s.io
-    version: v1alpha1
+    group: cert-manager.io
+    version: v1alpha2
     name: serving-cert # this name should match the one in certificate.yaml
   fieldref:
     fieldpath: metadata.namespace
 - name: CERTIFICATE_NAME
   objref:
     kind: Certificate
-    group: certmanager.k8s.io
-    version: v1alpha1
+    group: cert-manager.io
+    version: v1alpha2
     name: serving-cert # this name should match the one in certificate.yaml
 - name: SERVICE_NAMESPACE # namespace of the service
   objref:

